doc/guides/bluetooth/img/ble_cfg_dual.xml

```diff
@@ -0,0 +1 @@
+<mxfile modified="2019-03-07T14:48:10.055Z" host="www.draw.io" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/72.0.3626.121 Safari/537.36" version="10.3.5" etag="e-D3ClfASJyunx5gBdFe" type="dropbox"><diagram id="3d368da2-d055-8685-6f25-468c73dcbdd0" name="Page-1">7Vtbc+I2FP41zGwfkvEdeAQ2l52mncymmXb3pWOwAE2E5foSSH/9HtkSWJYCprG5dGFnFuvIlq3v3L+Yjj1arO5iP5r/RgNEOpYRrDr2545lmXbfgS8meSskXY8LZjEO+EkbwRP+F3GhwaUZDlAinZhSSlIcycIJDUM0SSWZH8d0KZ82pUS+a+TPkCJ4mvhElf6Jg3ReSHuusZHfIzybizubBp8Z+5OXWUyzkN+vY9nT/FNML3yxFj8/mfsBXZZE9k3HHsWUpsXRYjVChGErYCuuu31ndv3cMQrTOhfYXC+vPsmQeOT8wdI3AUa+HcQuMDr2cDnHKXqK/AmbXYL6QTZPFwRGJhwGfjLPz2UDn+BZCMcETeFphlNMyIgSGufrCmDsYZLG9AWVZvr5B2bU7fAdvqI4RauSiG/vDtEFSuM3OIXPOvwKbomW5Rbj5UavtpDNSzp1hbJ9bkuz9dIbPOGAQ/oOvK4GXo8wMAL8KsHs/ZMxpQ9hW+kVB24AZ+TYrWfhaMa/81WSyA+rsrEQfM58cjWZ46jwlCmeZbGfYgpXwJMb31E0f4vh4J4mqbgYdjSuLggy3X1AnG9CSCtmwzYi24as55CGqGIUXFQxG6ZrDL454OIFDgJ2E60pNmExfdlibFuxGMu1VIuxGjAY09QYTAVYiBkROyzwDWg2zmfM3c7pJ1ERLad4xZx0SPwxIo80wblVKIg/VKbXyAsNxcXu31dRA+ro9mR1dD2NOgyNOhrRh63oYxBFBPZa+JGxdrpYOMInNLtmCvMXEYEcBv4El0L2goeFY+SDJ/7SYIxtAOG+jLCtCZHrzFxGWITWDwGsJqDhw806KJ0yTD1NXGgNJm93XEBhMGC1D4zGhE5e8ojrx2lVWIIIwIjf/mJwXrti+E3MrXBamoLRN457vuotJptVgtKoiPKibLK3YY8CqQxTkS9Bq3NxIYsRAX98lYs3Hdz8Do8Uw5OsFWsZtqRZ13XkJRKaxRPErypXUzsWcvqevBAAN0OpslCu/fW26xlEVzGI+9EXEHx6Hnz9o0jwT49f1EhznLQs+WI160zALlBcJ+804MWWKbuxK8YlWzM9XT5pIJ0Ir1W0BpkUtlBDfacUAR3HOlwEtNRMrIBzTpWRHCatfdojAz63t61UV66ryf3a6spowh3U9mg+wX9nkF5ASpk/sHESYXajct11wl7hdg9YPllqXVCUTyMKuZwSwoLKCYPlWZpqvjWw1Jx5wG5carevaEgYKuMMk+DSe6/No1dp9jQ1tqkPRw2YR69l81BNYeOl+xjEz6D5riYwmL2WWBdbZV2KKPrVDzBlLuvHwRJ212AsrWZ7l/3TZXsv/7QTfeu2sFa/AZBrtLDnTjXv5podHbzNcM1HzW47uOa735/h+R9wmK06F8K5ajdeDca5tdiny3pn3Fc1oY/+MSlnu68oZHCqrY9bIVNs84C9j2MoQD0nKE7y3VvGEKTf2feJ0skKdt4BWyFHrXgUdC58srR4fT7ZteU64D/zydWFWuSTBZW3jU9+fhpe+GTFjQVyW/hkkU+azt2Oyon+iuIwfxmn0F6hvIJKA/XB/wFnmk85FDpuv14otLwGQKzxCsw5FUAnQiw74tWibcyyVqeG1YBS6zHLWTLunDCz7LgVDDXUstm1WyoRzo1arqKl45bbQ6vt7nvdJEut9M/HE7rGESli50IRH5MncWQH13LEoldvXPVqW/7/5IiVMKpzMF0YbYIkdvfrS7ldSqApdn1OrWOVenLEez77to5dZ8dCzbWO7pYuRJAwCxpk+duQp1QwuKbMuzp1/xjdRNvhvp9KpjTXpJpL2MRVkv9agqUS2P6qRiq5vr7ekS2KG55UujidBr/bq7A54sct5b/qaIxE9LQfMRJPpTi3hcMLTbdPrPX6DdF01YVapOk8XX68RI0ziBoaWlAXNZooVT1rd9S4MFp7q9Rzq6+Ja9gYUWZJmUD8iuRDStW9/7pf68kd4CPNp+cvmDmE4ySSg8HOJrSlRz1+M6yElUO1w57Xle2xpxYmfU2f5rl7WyMMN79JLdLW5oe/9s0P</diagram></mxfile>
```

Comment:
Thanks for including these drawing source files!  We may need to teach checkpatch not to worry about "misspellings" in XML files @nashif